src/webgpu/shader/validation/wgsl/variable-with-initilizer-storage.fail.wgsl

```diff
@@ -0,0 +1,13 @@
+# v-0032: variable 'u' has an initializer, however its storage class is 'storage'.
+
+[[block]]
+struct PositionBuffer {
+  [[offset(0)]] pos: vec2<f32>; 
+};
+
+[[group(0), binding(0)]]
+var<storage_buffer> s : PositionBuffer = {vec2<f32>(0.0, 0.0)}; 
+
```

Comment:
tint ouput:

variable-with-initilizer-storage.fail.wgsl:8:3 error: expected decoration
[[group(0), binding(0)]]
  ^^^^^

variable-with-initilizer-storage.fail.wgsl:9:42 error: unable to parse const literal
var<storage_buffer> s : PositionBuffer = {vec2<f32>(0.0, 0.0)};

Reply:
This is a Tint bug, we're still parsing `set` and not `group` yet.